apps/nextjs/src/env/schema.mjs

```diff
@@ -21,7 +21,7 @@ export const serverSchema = z.lazy(() =>
         process.env.VERCEL_URL
           ? `https://${process.env.VERCEL_URL}/api/auth`
           : str,
-      z.string().url()
+      process.env.VERCEL_URL ? z.string().url() : z.string() // zod's url throws on env var interpolation ($)
```

Comment:
@juliusmarminge, sorry I meant to suggest this as edits instead of pushing the commit directly. Still getting used to the in-borwser vscode on pull request reviews.

I'll post the actual discussion in the [relevant commit](https://github.com/agreeto/agreeto/pull/6/commits/23b90b95d6fa1ebf14266c82b21ce7bebb6c527a).